forge-gui/res/cardsfolder/f/false_dawn.txt

```diff
@@ -3,7 +3,7 @@ ManaCost:1 W
 Types:Sorcery
 A:SP$ Effect | Cost$ 1 W | ReplacementEffects$ FDRep | StaticAbilities$ FDManaConvertion | SubAbility$ DBDraw | SpellDescription$ Until end of turn, spells and abilities you control that would add colored mana instead add that much white mana. Until end of turn, you may spend white mana as though it were mana of any color. Draw a card.
 SVar:DBDraw:DB$ Draw | NumCards$ 1
-SVar:FDRep:Event$ ProduceMana | ActiveZones$ Command | ValidAbility$ SpellAbility.YouCtrl | ReplaceWith$ ProduceW | Description$ Spells and abilities you control that would add colored mana add that much white mana instead.
+SVar:FDRep:Event$ ProduceMana | ActiveZones$ Command | ValidAbility$ SpellAbility.YouCtrl | NonManaAbility$ True | ReplaceWith$ ProduceW | Description$ Spells and abilities you control that would add colored mana add that much white mana instead.
```

Comment:
Why does this exclude mana abilities?

Reply:
No, this is the only card that should include them, all others don't work for Spells etc.:
> 106.12b. A replacement effect that applies if a permanent "is tapped for mana" or tapped for mana of a specific type and/or amount modifies the mana production event while such an ability is resolving and producing mana or the specified type and/or amount of mana.

Reply:
But wouldn't your change stop this from working with mana abilities?

I'm pretty sure the card should work on mana abilities

Reply:
Works as expected, the param enables it for nonmana.

Example:
_Deathrite Shaman + Ashaya_ = _Naked Singularity_ in play only replaces the granted from Forest, the Deathrite can still produce any with his own
If _False Dawn_ resolved instead both would only produce {W}

Reply:
But it should be work on mana abilities too

There is nothing telling on False Dawn doesn't work on mana abilities

Reply:
Yea I agree and that works
with False Dawn all lands etc. tap for {W}

Reply:
Should be ok to merge?

Reply:
imo NonManaAbility should be removed

Reply:
- If I remove it _False Dawn_ can no longer replace mana from spells, that's wrong
- If I remove the `isManaAbility()` check in `ReplaceProduceMana` completely cards like _Naked Singularity_ would also replace nonmana like of _Deathrite Shaman_, that's also wrong
- the other solution would be to update such other cards with `ValidAbility$ Activated.hasTapCost**+ManaAbility**` instead - is that what you'd prefer?

Reply:
but currently, `NonManaAbility$ True` would stop it working with ManaAbilities, and thats also wrong?

Reply:
nooo, I'm telling you it means Include "non" not exclude:
![image](https://user-images.githubusercontent.com/8506892/203014879-a4c786de-f65f-4557-b519-1f16d2910ab3.png)

Reply:
we probably need to update later what `NonManaAbility` means at this point

we can merge this for now, but i think that might need some cleanup later.

IMO `Activated.hasTapCost+ManaAbility` would be good too